src/_cffi_src/openssl/ssl.py

```diff
@@ -470,17 +470,19 @@
     return 1;
 }
 #endif
+
 /* Added in 1.0.2 but we need it in all versions now due to the great
    opaquing. */
-#if OPENSSL_VERSION_NUMBER < 0x10002001L || defined(LIBRESSL_VERSION_NUMBER)
```

Comment:
Do you know of a specific platform impacted here? I'm reluctant to add more conditions for a theoretical case, given that all the CA passes.